### PR TITLE
[openscad/en] Add OpenSCAD learnxinyminutes

### PR DESCRIPTION
**Pull request by tompreston**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [N/A] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by tompreston** [comment]
I fixed up the filename and the formatting secion.

**Comment by tompreston** [comment]
Hey @mribeirodantas, I noticed you were merging things recently. Are the maintainer?

If you have some time, I'd really appreciate a review 🙏 

Cheers

File: openscad.html.markdown [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+language: openscad
+filename: leanopenscad.scad
```
Comment (mribeirodantas) [inline openscad.html.markdown:3]:
```suggestion
filename: learnopenscad.scad
```

File: openscad.html.markdown [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    - ["Thomas Preston", "https://github.com/tompreston/"]
+---
+
+Draw 3D models with code using [OpenSCAD](https://openscad.org/).
+
+```openscad
+// Single-line comments start with //
+
+/*
+Multi-line comments look like this.
```
Comment (mribeirodantas) [inline openscad.html.markdown:14]: In the other multi-line comments, you have a * at the beginning of the line. Is this optional?

**Review by mribeirodantas — APPROVED** [review]

File: openscad.html.markdown (hunk 2 above)
Comment (tompreston) [reply openscad.html.markdown]: [It's optional](https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/General#Comments)

In retrospect, it's a bit confusing having the two kinds of comments at all. I've removed the multi-line comments completely.

**Comment by mribeirodantas** [comment]
Thanks for your contribution, Tom 😄

**Comment by ShiftWatchOut** [comment]
Got problem with accessing [openscad](https://learnxinyminutes.com/docs/openscad/) documentation on the site.

**Comment by nbehrnd** [comment]
Intermediate by-pass: checkout the repository on GitHub. With all the logs and
history, this archive is about 40 MB (or fetch the bare .zip, about 8MB) while
the markdown files often are less than 10KiB; meanwhile, the authors may work
on a solution.

**Pull request merged**